### ISSUE TITLE
Google Pay app does not works

### ISSUE THREAD

**Issue by Shished** [wacko1805/MagiskGapps#35]
Google Pay app does not works with MagiskGapps. Sometimes it just crashes after opening, other times it shows "Can't set active account" message.
Tested on 2 phones with LineageOS 20 and 21, used MindTheGapps for android 14 (20240925) and for android 13 (20231025).

**Comment by wacko1805** [comment]
Have you logged into your Google account on the device?

**Comment by wacko1805** [comment]
If so, Have you made sure the the Google Pay app and Google Play Services apps are updated to the latest versions?

**Comment by Shished** [comment]
Yes i've logged in the account and Play Store works, i installed Google Pay app from it. Play services app also got updated after the installation.

**Comment by wacko1805** [comment]
Thanks, will try to replicate the issue and find a fix.

**Comment by Shished** [comment]
Interestingly, after just letting the phone sit for about two days, Google Wallet began working correctly without me doing anything further. I noticed the exact same pattern with Google Quick Share – it also didn't work right after the install but started functioning properly..

My suspicion is that this might be related to some background GMS update that needs time to complete after the initial installation.

I did try manually sideloading a newer GMS Core APK while Wallet and Quick Share were still non-functional, but that didn't seem to fix the issue.
